test/basic.html

```diff
@@ -118,7 +118,7 @@
 						done();
 					}, 200);
 
-					suggestions._input.fire('focus', {});
+					suggestions.dispatchEvent(new CustomEvent('focus', { bubbles: true, composed: true }));
```

Comment:
Same as above

Reply:
No, this was a mistake that also halted the tests. Added it back.